PyQt5-stubs/QtTest.pyi

```diff
@@ -151,9 +151,11 @@ class QTest(sip.simplewrapper):
     @staticmethod
     def mouseClick(window: QtGui.QWindow, button: QtCore.Qt.MouseButton, modifier: typing.Union[QtCore.Qt.KeyboardModifiers, QtCore.Qt.KeyboardModifier] = ..., pos: QtCore.QPoint = ..., delay: int = ...) -> None: ...
     @typing.overload
-    def keySequence(self, widget: QtWidgets.QWidget, keySequence: typing.Union[QtGui.QKeySequence, QtGui.QKeySequence.StandardKey, str, int]) -> None: ...
```

Comment:
I think the two already staticmethods should be removed. There's no static version according to the Qt docs:
https://doc.qt.io/qt-5/qtest.html#keySequence